packages/eslint-plugin-turbo/lib/utils/getEnvVarDependencies.ts

```diff
@@ -1,75 +0,0 @@
-import { getTurboConfigs } from "@turbo/utils";
-
-function findDependsOnEnvVars({
-  dependencies,
-}: {
-  dependencies?: Array<string>;
-}) {
-  if (dependencies) {
-    return (
-      dependencies
-        // filter for dep env vars
-        .filter((dep) => dep.startsWith("$"))
-        // remove leading $
-        .map((envVar) => envVar.slice(1, envVar.length))
-    );
-  }
-
-  return [];
-}
-
-function getEnvVarDependencies({
-  cwd,
-}: {
-  cwd: string | undefined;
-}): Record<string, Set<string>> | null {
-  const turboConfigs = getTurboConfigs(cwd);
-
-  if (!turboConfigs.length) {
-    return null;
-  }
-
-  const envVars: Record<string, Set<string>> = {
-    "//": new Set(),
-  };
-
-  turboConfigs.forEach((turboConfig) => {
-    const { config, workspacePath, isRootConfig } = turboConfig;
-
-    const key = isRootConfig ? "//" : workspacePath;
```

Comment:
This was a sneaky bug. We only supported workspaces correctly if they had a `turbo.json`.

New edition removes that limitation by using the new `getWorkspaceConfigs`.